packages/ember-metal/lib/error_handler.js

```diff
@@ -13,6 +13,12 @@ let getStack = error => {
   return stack;
 };
 
+export const onErrorTarget = {
+  get onerror() {
+    return dispatchError || getOnerror();
```

Comment:
Maybe I'm missing something, but I don't see how this works since `dispatchError` is always defined just below here?

I would think you want this to actually be:

```js
export const onErrorTarget = {
  get onerror() {
    // will need to be moved to below where these are defined (e.g. down to L41 below)
    return dispatchOverride || onerror;
  }
}
```